### PR TITLE
Update html.parser.rst

### PR DESCRIPTION
**Pull request by geryogam**
This PR for the documentation of the [html.parser](https://docs.python.org/3/library/html.parser.html) module will

- fix a terminology mistake (‘start of a tag’ -> ‘start tag of an element’);
- mention the parameter names of the `HTMLParser.handle_*` methods where missing.

Automerge-Triggered-By: GH:merwok

### CODE REVIEW

File: Doc/library/html.parser.rst
Comment (JelleZijlstra) [inline Doc/library/html.parser.rst]: Why is it necessary to mention the parameter name explicitly? The old wording is a little more concise and doesn't seem ambiguous to me. This applies to most of your other changes too.

Comment (geryogam) [reply Doc/library/html.parser.rst]: Because the [page](https://docs.python.org/3/library/html.parser.html) already mentions parameter names for some methods:

- `HTMLParser.handle_decl(decl)`:

  > The *decl* parameter will be the entire contents of the declaration inside the `<!...>` markup (e.g. `'DOCTYPE html'`).

- `HTMLParser.handle_pi(data)`:

  > The *data* parameter will contain the entire processing instruction.

- `HTMLParser.unknown_decl(data)`:

  > The *data* parameter will be the entire contents of the declaration inside the `<![...]>` markup.

Comment (JelleZijlstra) [reply Doc/library/html.parser.rst]: OK. I'll leave this up to the maintainer of the module, but personally I don't think consistency is too important here.

File: Doc/library/html.parser.rst
Comment (merwok) [inline Doc/library/html.parser.rst]: Is this gloss necessary?  People reading this page should know that much about python functions.

File: Doc/library/html.parser.rst
Comment (merwok) [reply Doc/library/html.parser.rst]: Seems unneeded to me too.

File: Doc/library/html.parser.rst
Comment (merwok) [inline Doc/library/html.parser.rst:129]: IMO this change is correct and useful, but the other changes are rewordings that do not make the text more exact or useful.

Comment (geryogam) [reply Doc/library/html.parser.rst:129]: I have just removed the other consistency changes and kept this fix.

**Review by JelleZijlstra — APPROVED** [review]

**Review by merwok — APPROVED** [review]

**Comment by merwok** [comment]
I should have changed the PR title to get a better commit message, like «Improve wording in html.parser doc»

«update file x» is not very useful, we try to always explain what is done (and often why too).

**Comment by maggyero** [comment]
Thanks for the review @merwok, and next time I will choose a better PR title than the default one.

**Pull request merged**
